src/flash/flasher.cpp

```diff
@@ -12,15 +12,14 @@
 PING_LOGGING_CATEGORY(FLASH, "ping.flash")
 
 Flasher::Flasher(QObject* parent)
-    : QObject(parent)
-{
-    _binRelativePath =
+    : QObject(parent),
+    _binRelativePath(QCoreApplication::applicationDirPath()
```

Comment:
use comma in the next line to make it better to read and avoid future unnecessary diffs.
```
: blah
, foo
, bee
```

Reply:
this is really bad to read :)